src/Smalot/PdfParser/Element/ElementHexa.php

```diff
@@ -47,12 +47,11 @@ public static function parse(string $content, ?Document $document = null, int &$
         if (preg_match('/^\s*\<(?P<name>[A-F0-9]+)\>/is', $content, $match)) {
             $name = $match['name'];
             $offset += strpos($content, '<'.$name) + \strlen($name) + 2; // 1 for '>'
-            // repackage string as standard
-            $name = '('.self::decode($name).')';
-            $element = ElementDate::parse($name, $document);
+            $name = self::decode($name);
+            $element = ElementDate::parse('('.$name.')', $document);
 
             if (!$element) {
-                $element = ElementString::parse($name, $document);
+                $element = new self($name);
             }
```

Comment:
Please give me a short summary what you try to accomplish here.

Why removing the reference to `$document`?

Reply:
Also why switching from `ElementString` to `ElementHexa`? Is it to match others existing behavior from other elements classes?